### DISCUSSION TITLE
Centre Text and File Names

### DISCUSSION THREAD

**Discussion by ProStashio** [RafaelGB/obsidian-db-folder#790] [Q&A]
### What happened?

When a row becomes taller, file names and text do not become centred like other sections.
<img width="730" alt="d7405dbde8d8962084441f087db0709e" src="https://user-images.githubusercontent.com/19267479/215493173-aab6d200-319f-4aa9-bb83-b8774895da13.png">

### What platform were you using?

Desktop

### Version of the plugin

Current

### Are you using the latest version of the plugin in your Obsidian vault?

- [X] I have verified that I am on the latest version

### Are you check if there is a similar issue?

- [X] I have verified that there are not similar issues

**Comment by RafaelGB** [comment]
You can choose the style of your column align

<img width="413" alt="choose options" src="https://user-images.githubusercontent.com/11924043/215503710-442a826a-3d2b-4455-ae97-267396c1498c.png">

**Comment by ProStashio** [comment]
I have that selected, I have the same issue
![image](https://user-images.githubusercontent.com/19267479/215506843-c5e8cdea-43a5-4869-b0f5-f70f8bb5dd5a.png)

**Reply by RafaelGB** [reply]
Which theme are you using?  I can not reproduce the issue with default or Minimal ones

**Reply by ProStashio** [reply]
ITS Theme

**Reply by RafaelGB** [reply]
Looks like its a theme problem. The issue does not happen with another ones

https://github.com/RafaelGB/obsidian-db-folder/blob/d3ea5b63d1728ce09528846ade4ce4a8d9f4368c/styles.css#L672

It should have been snipping this css

**Reply by ProStashio** [reply]
Is there a fix on my end?

**Reply by RafaelGB** [reply]
You can those lines as css snippet:
```css
.database-plugin__align-top {
  align-items: flex-start;
}

.database-plugin__align-middle {
  align-items: center;
}

.database-plugin__align-bottom {
  align-items: flex-end;
}
```

I don't know the content of the ITS theme. You can open an issue in its repo relating this one
